test/Libraries/Microsoft.Extensions.Telemetry.Tests/Buffering/GlobalBufferLoggerBuilderExtensionsTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#if NET9_0_OR_GREATER
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Xunit;
+
+namespace Microsoft.Extensions.Logging.Test;
+
+public class GlobalBufferLoggerBuilderExtensionsTests
+{
+    [Fact]
+    public void AddGlobalBuffer_RegistersInDI()
+    {
+        var serviceCollection = new ServiceCollection();
+        serviceCollection.AddLogging(builder =>
+        {
+            builder.AddGlobalBuffer(LogLevel.Warning);
+        });
+
+        var serviceProvider = serviceCollection.BuildServiceProvider();
+        var buffer = serviceProvider.GetService<ILoggingBuffer>();
```

Comment:
I'm not sure we'd want this buffer registered under a public interface in DI. Is there a user-scenario where it matters?

Reply:
Updated